styles/style.css

```diff
@@ -0,0 +1,81 @@
+.container {
+  background-color: #e5e5e5;
+  width: 389px;
```

Comment:
[OPTIONAL] & [HIGHLY RECOMMENDED]
Because we are building using the mobile first approach, and still because the mobiles can have different size screens, it is highly recommended that you are not using fix widths for your containers:

 **How your design looks on a bigger mobile size screen:**
![image](https://user-images.githubusercontent.com/83596135/163899775-dfd224aa-5758-4ff5-a064-00f679376679.png)

**How it could look on the same screen size:**
![image](https://user-images.githubusercontent.com/83596135/163899953-faaf4f93-7a27-4606-b911-ae10ff43511e.png)

**Hint:**
for it to look like this, I removed the fix-widths in the CSS file from:
- `.container`
- `main nav`
- `main`
- `h1`
- `main p`
**As well, I added to `main` the following: `background-size: 100vw;`